src/Pages/HomePage/Homepage.jsx

```diff
@@ -50,27 +40,21 @@ const Homepage = () => {
 
   const updatehandler = async (e, editnotes) => {
     e.preventDefault();
-    console.log(editnotes);
-    try {
-      const resp = await axios.post(
-        `/api/notes/${editnotes._id}`,
-        { note: editnotes },
-        {
-          headers: {
-            authorization: encodedToken,
-          },
-        }
-      );
-      noteDispatch({
-        type: "UPDATE_NOTE",
-        payload: resp.data.notes,
-      });
-      setNotes(initialNotes);
-    } catch (error) {
-      console.log(error.message);
-    }
+    const resp = await editNotesService(editnotes, encodedToken);
+    noteDispatch({
+      type: "UPDATE_NOTE",
```

Comment:
I haven't implemented APIs in my macro project yet but after doing few PR reviews, I observed that you can add the dispatch function in your services itself. So just look up if its possible to add this dispatch in your services

Reply:
sure i will do this